src/common/lib/client/realtimechannel.ts

```diff
@@ -795,6 +798,11 @@ class RealtimeChannel extends Channel {
     );
     this.clearStateTimer();
 
+    // RTP5a1
+    if (['detached', 'suspended', 'failed'].includes(state)) {
```

Comment:
Same goes for `Array.includes` wrt IE. We have a `Utils.arrIn` helper for this.

Reply:
thanks updated